doc/source/train/user-guides/results.rst

```diff
@@ -0,0 +1,91 @@
+Inspecting Training Results
+===========================
+
+The return value of your :meth:`Trainer.fit() <ray.train.trainer.BaseTrainer.fit>`
+call is a :class:`~ray.air.result.Result` object.
+
+The :class:`~ray.air.result.Result` object contains, among others:
+
+- The last reported metrics (e.g. the loss)
+- The last reported checkpoint (to load the model)
+- Error messages, if any errors occurred
+
+Last reported metrics
```

Comment:
How about grouping "Viewing metrics" as one header and having sub-headers for latest/all metrics?

That way we can add some general context on what to do with metrics at the top level, that is applicable to all.